spec/pb/ThingService.proto

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2020-present, BigCommerce Pty. Ltd. All rights reserved
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+// persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+// Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+// WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+// COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+// OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+syntax = "proto3";
+
+package gruf.demo;
+
+// Demonstration service
+service ThingService {
```

Comment:
I think we are using this a couple apps now. Do you think there is value in creating a `gruf-demo-thing-service` gem and maybe include some rspec helpers to make the test setup a bit easier?

Reply:
Maybe. May be better to just include this, and other helpers, in `gruf-rspec` more generally.